modyn/protos/storage.proto

```diff
@@ -66,6 +68,14 @@ message GetDataPerWorkerResponse {
   repeated int64 keys = 1;
 }
 
+message GetDatasetSizeRequest {
+  string dataset_id = 1;
+}
+
+message GetDatasetSizeResponse {
+  int32 num_keys = 1;
```

Comment:
this should be int64. For example with Criteo, these can be billions of samples. probably even uint64_t + a boolean success field instead

Reply:
Done